main.cpp

```diff
@@ -20,95 +20,189 @@ You are going to write 10 UDTs in project3.
+Thing 1) Coding Language
+5 properties:
+    1) number of data types
+    2) number of function/return types
+    3) number of operators
+    4) number of methods of punctuation
+    5) speed of compiler - bits/sec
+3 things it can do:
+    1) create functions and run functions from libraries
+    2) create data structures to store info
+    3) compile to machine code
  */
 
 /*
-Thing 1)
+Thing 2) Recording Studio
 5 properties:
-    1)
-    2)
-    3)
-    4)
-    5)
+    1) number of channels on console
+    2) number of compressors
+    3) number of time-based effects units
+    4) number of monitoring options
+    5) size of live room - square feet
 3 things it can do:
-    1)
-    2)
-    3)
+    1) Record music
+    2) Mix music
+    3) Host concerts
  */
 
 /*
-Thing 2)
+Thing 3) Electric Guitar
 5 properties:
-    1)
-    2)
-    3)
-    4)
-    5)
+    1) number of strings
+    2) number of frets
+    3) number of pick ups
+    4) number of pick up positions
+    5) number of knobs
 3 things it can do:
-    1)
-    2)
-    3)
+    1) amplify sound from string vibrations
+    2) produce a warm tone
+    3) produce a bright tone 
  */
 
 /*
-Thing 3)
+Thing 4) Refrigerator/Freezer
 5 properties:
-    1)
-    2)
-    3)
-    4)
-    5)
+    1) number of shelves
+    2) number of drawers
+    3) temperature level value - celsius
+    4) size of ice-machine - volume
+    5) size of entire-unit - volume
 3 things it can do:
-    1)
-    2)
-    3)
+    1) chill food
+    2) freeze food
+    3) separate food
  */
 
 /*
-Thing 4)
+Thing 5) struct MidiPads
 5 properties:
-    1)
-    2)
-    3)
-    4)
-    5)
+    1) velocity sensitive
+    2) hardness of rubber
+    3) size of pad
+    4) number of pads
+    5) pads' midi value
 3 things it can do:
-    1)
-    2)
-    3)
+    1) trigger sampler
+    2) send note data
+    3) send velocity data
  */
 
+/*
+Thing 6) struct TransportButtons
+5 properties:
+    1) record button
+    2) play button
+    3) stop button
+    4) fast forward button
+    5) rewind button
+3 things it can do:
+    1) start session audio
+    2) pause session audio
+    3) reset play head position to Bar 0
+ */
 
+/*
+Thing 7) struct Screen
+5 properties:
+    1) brightness
+    2) hue
+    3) size
+    4) Boarder perimeter width
+    5) touch sensitivity
+3 things it can do:
+    1) Scroll 
+    2) click via touch
+    3) control fx via touchbars sending midi cc's
+ */
+
+/*
+Thing 8) struct AudioIO
+5 properties:
+    1) analog to digital conversion
+    2) digital to analog conversion
+    3) sample rate
+    4) buffer size
+    5) audio file type
+3 things it can do:
+    1) convert analog signal to digital signals
+    2) convert digital signal to analog signals
+    3) write converted analog signals to storage
+ */
+
+/*
+Thing 9) struct AudioApplication
+5 properties:
+    1) compressor
+    2) sampler
+    3) reverb
+    4) digital synthesizer
+    5) pitch shifter
+3 things it can do:
```

Comment:
These are objects, not primitives!   You need to pick things that can be represented with `int float double bool char std::string`
- [ ] revise